test/test_jit.py

```diff
@@ -406,7 +408,9 @@ def test_backward_opaque(self):
 
         # Run dead code elimination to remove unused trace nodes
         torch._C._jit_pass_dce(trace)
-        self.assertExpected(str(trace))
+        # This is nondeterministic, see:
+        #   https://github.com/ezyang/pytorch/issues/227
+        #self.assertExpected(str(trace))
```

Comment:
Skip this test explicitly then. Well easily forget that it’s broken otherwise

Reply:
I think there is still value in exercising the code (even if we don't actually then check if the output is what we expect.)

Reply:
You can use SkipError to skip from the end of the test (and if sth fails before it will be a failure)